selfrole.py

```diff
@@ -137,3 +139,90 @@ async def apply_roles(self, ctx: commands.Context, roles: set):
                 embed.colour = 0xF04747
             embed.add_field(name="Invalid roles", value="\n".join(errors), inline=False)
         await ctx.send(embed=embed)
+
+    @commands.group(name="rolestats", enabled=opt.enable_role_stats)
+    @commands.guild_only()
+    async def _role_stats(self, ctx: commands.Context):
+        """Show statistics for roleme-managed roles."""
+        if ctx.invoked_subcommand is None:
+            role_stats: Dict[discord.Role, int] = {}
+
+            if ctx.guild:

```

Comment:
What is that one solving?
Getting the guild should always be possible, unless the cache settings are not correct.

Reply:
mypy yells because `ctx.guild` is technically optional

Reply:
This is a case of "shut-up mypy"
Mypy is not run on the repo yet, so let's ignore it.

Later we'll have to find how to tell it to shut up when the condition is impossible.